Confus/Confus/Flag.cpp

```diff
@@ -209,13 +147,46 @@ namespace Confus
         m_StartRotation.set(a_Rotation);
     }
 
+    void Flag::setPosition(irr::core::vector3df a_Position)
+    {
+        m_FlagNode->setPosition(a_Position);
+    }
+
     void Flag::returnToStartPosition() {
         m_FlagNode->setParent(m_FlagOldParent);
         m_FlagNode->setPosition(m_StartPosition);
         m_FlagNode->setRotation(m_StartRotation);
 		setFlagStatus(EFlagEnum::FlagBase);
     }
 
+    void Flag::setConnection(Networking::ClientConnection* a_Connection)
+    {
+        m_Connection = a_Connection;
```

Comment:
Can you handle this externally possibly? Keep in mind that the code for the Flag will be shared between the server and client. I'm fine with taking that task as well